add-pkg.sh

```diff
@@ -20,7 +22,7 @@ if test "$TYPE" == "r"; then
 else
     docker run --name "$NAME" --entrypoint bash "$IMAGE" -c "apt-get install -y $PACKAGE"
 fi
-docker commit --change "CMD []" --change 'ENTRYPOINT ["/usr/bin/Rscript"]' "$NAME" "r-$NAME"
+docker commit --change "CMD []" --change 'ENTRYPOINT ["/usr/bin/Rscript"]' --change "LABEL org.opencontainers.image.created=$BUILD_DATE org.opencontainers.image.revision=$REVISION" "$NAME" "r-$NAME" 
```

Comment:
I think we gained a tailing whitespace on this line if I'm reading the diff right… does that matter? Ignore if not 🙂